src/client/useActions.ts

```diff
@@ -102,18 +79,27 @@ export const useActions = (
     [dispatch],
   );
 
+  // New action function for setting the tab list
+  const setTabList = useCallback(
+    (tabList: Array<TabState>) => {
+      dispatch({
+        type: 'set_tab_list',
+        tabList,
+      });
+    },
+    [dispatch],
+  );
+
   return {
     setActiveFileId,
-    setActiveFileLeft,
```

Comment:
These are used in the keyboard shortcuts. Deleting them will break things.

Reply:
Before deleting things, it's a good idea to search for them in the codebase to see if they are used anywhere.